wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenList.tsx

```diff
@@ -63,21 +64,24 @@ const TokenList = (props: Props) => {
   );
 
   const sortedTokens = useMemo(() => {
-    const { selectedToken, selectedChainConfig } = props;
```

Comment:
If we refer to props like this, exhaustive deps requires it to be in the list; which we don't want to.